### ISSUE TITLE
composer.json should list symfony versions

### ISSUE THREAD

**Issue by tacman** [975L/SiteBundle#5]
Instead of symfony/framework-bundle: "*", it'd be better if you explicitly listed which versions this bundle works with, e.g. ^6.4||^7.0.

Also, the documentation refers to the old style templating, the modern Symfony structure puts templates in the /templates directory, rather that Resources/views.

**Comment by LaurentMarquet** [comment]
Yes, you're right, but as I'm the only maintainer and mainly only user, and missing time, I haven't updated data...
